db/migrate/20220429000930_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] <b>REQUIRED:</b> And here too, Kindly make sure that the foreign key for the user in the comments table is named `author_id` instead of `user_id`.

Reply:
Hi thanks for the review, Kindly ask you to share with me the link of how to change this because following the milestone resources they teach to do it like that